content/en/community/contributing/_index.md

```diff
@@ -0,0 +1,214 @@
+
+The CHT is [open source](https://en.wikipedia.org/wiki/Open_source). Any code you commit will be freely available for other CHT users and they can improve and customize it just as you might choose to do so with the code others have written.
+
+### How code gets written
+
+Over the years the process for writing the CHT has been codified in our [workflow]({{% ref "community/contributing/code/workflow" %}}).  Skim this so you're aware of the basics. Check out our code  [style guide]({{% ref "community/contributing/code/style-guide" %}}) as well. You don't have to memorize everything here, but it's important to know about and refer back to when it the time comes to actually writing code.
+
+A good rule of thumb is mimic the processes you see and the styles already present in the source code.
+
+If you have never used `git` or GitHub, checkout the [Get started using GitHub in less than an hour](https://github.com/skills/introduction-to-github) guide. This will help you set up a GitHub account which you'll need as well. If you haven't worked on a pull request before, check out [How to Contribute to an Open Source Project](https://egghead.io/lessons/javascript-introduction-to-github) and [How to Contribute to Open Source](https://firstcontributions.github.io/contribute-to-opensource/).
+
+### Your first issue
+
+Find an issue marked [Good first issue](https://github.com/medic/cht-core/issues?q=is%3Aopen%20is%3Aissue%20label%3A%22Good%20first%20issue%22). Feel free to pick an issue that looks easy for you! An easy looking issue will better set you up for successfully merging the code. Easy merges is our goal here!  
+
+Make a comment on the issue asking for the issue to be assigned to you.
+
+**Note** -  [Help Wanted](https://github.com/medic/cht-core/issues?q=is%3Aopen%20is%3Aissue%20label%3A%22Help%20Wanted%22) issues do not have the extra introduction that Good first issues have.  While you're welcome to work on them, they'll be more of a challenge for first time contributors that likely will want some guidance found in the Good first issues.
+
+### Fork the CHT
+
+Good work! if you've gotten this far you understand what the CHT is and you know how code is written for the CHT. Now it's time to set up your very own copy of the CHT, called a fork.  
+
+Follow these steps to set up a copy of the code ("fork") which you have on your computer ("git clone"). _If you're not comfortable with command line, [see tutorials using GUI tools.](#tutorials-using-other-tools)_
+
+
+If you do not have a Github account, [please create one now](https://github.com/signup) as it's a requirement to fork and contribute code.
+
+
+1. Create a Fork
+
+    Fork this repository by clicking on the fork button on the top of this page.
```

Comment:
It's unclear to me what "this repository" means. Is this cht-core? or any CHT repo?

Reply:
yes - good point!  changed to:

> Create your own fork the CHT Core repository by going to the CHT's [fork page on GitHub](https://github.com/medic/cht-core/fork). When you click "Create Fork" on that page, you will  create a copy of this repository in your GitHub account.

and below updated: 

> where "url you just copied" (without the quotation marks) is the url to the repository (your fork of the CHT Core). See the previous steps to obtain the url.

see 5d90eec and 73b2659